src/main/java/com/tenx/ms/retail/converter/OrderConverter.java

```diff
@@ -0,0 +1,42 @@
+package com.tenx.ms.retail.converter;
+
+
+import com.tenx.ms.retail.domain.OrderEntity;
+import com.tenx.ms.retail.domain.StoreEntity;
+import com.tenx.ms.retail.rest.dto.Order;
+
+public final class OrderConverter {
```

Comment:
it would be better to use @Component & @Autowired instead of static singleton class with static method